### ISSUE TITLE
turn_animation_into_updater(anim) calls anim.update_config(**kwargs) which was removed

### ISSUE THREAD

**Issue by sparshg** [ManimCommunity/manim#951]
## Description of bug / unexpected behavior

Calling `turn_animation_into_updater(some_animation)` _which is in "mobject/mobject_update_utils.py"_ will give error because it is calling `update_config()` method from Animation class but it was removed in previous release

## How to reproduce the issue

```py
triangle = Triangle()
anim = Write(triangle, run_time=1)
self.add(triangle)
turn_animation_into_updater(anim)
```

## Logs
```
Traceback (most recent call last):
  File "/home/user/.local/lib/python3.9/site-packages/manim/__main__.py", line 95, in main
    scene.render()
  File "/home/user/.local/lib/python3.9/site-packages/manim/scene/scene.py", line 165, in render
    self.construct()
  File "scene.py", line 7, in construct
    turn_animation_into_updater(anim)
  File "/home/user/.local/lib/python3.9/site-packages/manim/mobject/mobject_update_utils.py", line 86, in turn_animation_into_updater
    animation.update_config(**kwargs)
AttributeError: 'Write' object has no attribute 'update_config'
```

**Comment by behackl** [comment]
Thanks for reporting! This is a consequence of the removal of CONFIG dicts. Were there any other `update_config` calls, @cobordism @leotrs -- and if so, can you remember what you did with them?

**Comment by friedkeenan** [comment]
The method was actually removed in #881. According to [this comment](https://github.com/ManimCommunity/manim/pull/881#discussion_r549462863), it was only used in one place, but I guess it turns out it wasn't.

**Comment by sparshg** [comment]
```bash
site-packages/manim » grep -r update_config *

animation/transform.py:    def update_config(self, **kwargs: typing.Dict[str, typing.Any]) -> None:

animation/transform.py:        Animation.update_config(self, **kwargs)
grep: animation/__pycache__/transform.cpython-39.pyc: binary file matches

mobject/mobject_update_utils.py:    animation.update_config(**kwargs)
grep: mobject/__pycache__/mobject_update_utils.cpython-39.pyc: binary file matches
```
There's something in animation/transform.py you might want to look

**Comment by behackl** [comment]
Indeed; my bad. In that case, sorry for the wrong ping. Nevertheless, replacing the calls to `update_config` by something like
```py
for attr, value in kwargs.items():
    setattr(animation, attr, value)
```
as suggested by @friedkeenan in the comment over at #881 should be a rather straightforward solution for this. We just might need to pull the extra code for `"path_arc"` from `transform.py` to `animation.py`, with an `isinstance` check or so...

**Comment by sparshg** [comment]
This bug was fixed in #1241
